### PR TITLE
add pod ready time metrics

### PR DESCRIPTION
**Pull request by superbigsea**
**What this PR does / why we need it**:
add pod ready time metrics with test case 
**How does this change affect the cardinality of KSM**: *(increases, decreases or does not change cardinality)*

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: internal/store/pod_test.go [hunk 1]
```diff
@@ -1132,18 +1132,25 @@ func TestPodStore(t *testing.T) {
 						{
 							Type:   v1.PodReady,
 							Status: v1.ConditionTrue,
+							LastTransitionTime: metav1.Time{
+								Time: time.Unix(1501666018, 0),
+							},
+
```
Comment (dgrisonnet) [inline internal/store/pod_test.go:1138]: This empty line should be removed for the lint job to succeed.

**Review by dgrisonnet — COMMENTED** [review]
You will need to update the number of expected metrics to 52 in https://github.com/kubernetes/kube-state-metrics/blob/master/internal/store/pod_test.go#L1751 since you are adding a new metric.

Other than that, looks good to me on green CI.

File: internal/store/pod.go [hunk 2]
```diff
@@ -1568,6 +1569,31 @@ func createPodStatusUnschedulableFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodReadyTimeFamilyGeneratorFamilyGenerator() generator.FamilyGenerator {
```
Comment (Serializator) [inline internal/store/pod.go:1572]: **nitpick;** I'd suggest renaming it to `createPodStatusReadyTimeFamilyGenerator` to keep it consistent with others such as `createPodStatusScheduledTimeFamilyGenerator`.

File: internal/store/pod_test.go [hunk 3]
```diff
@@ -1132,18 +1132,25 @@ func TestPodStore(t *testing.T) {
...
 						},
 					},
 				},
 			},
 			Want: `
 				# HELP kube_pod_status_ready Describes whether the pod is ready to serve requests.
-				# TYPE kube_pod_status_ready gauge
```
Comment (Serializator) [inline internal/store/pod_test.go:1141]: **nitpick;** the identation seems a bit of.

File: internal/store/pod_test.go [hunk 4]
```diff
@@ -1163,10 +1170,12 @@ func TestPodStore(t *testing.T) {
...
```
Comment (Serializator) [inline internal/store/pod_test.go:1166]: **nitpick;** idem, the identation which seems a bit of.

**Pull request closed without merging**